src/subcommand/torrent/create.rs

```diff
@@ -240,6 +240,11 @@ Sort in ascending order by size, break ties in descending path order:
             download and upload statistics to multiple trackers."
   )]
   source: Option<String>,
+  #[structopt(
+    long = "ignore-files",
```

Comment:
I opted for 'ignore-file*s*' instead of the singular 'ignore-file', as the `ignore` crate supports ignoring files listed in '.gitignore' _and_ '.ignore' files. If you'd rather have that ignore functionality split into 2 different flags, I'll do that.

Reply:
I think one flag is good, but `--ignore-files` makes me think that maybe it takes an argument. What about just `--ignore`, which is what ripgrep uses?